components/RecipeList.js

```diff
@@ -21,6 +27,11 @@ export default function RecipeList() {
           </Link>
         </li>
       ))}
+      <li key="createLink">
```

Comment:
```suggestion
      <li>
```

you only need to set a key if you are dynamically mapping list-elements